ballerina/build.gradle

```diff
@@ -84,7 +84,9 @@ task clean {
 }
 
 task build {
-    build.dependsOn ':delivery-ballerina:delivery:build'
+    build.dependsOn ':delivery-ballerina:api_del_doc_with_credit_block:build'
+    build.dependsOn ':updateDocumentationFiles'
```

Comment:
https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.procurement/blob/main/ballerina/build.gradle#L88

No : in the front